### PR TITLE
support for gecko tool

### PR DESCRIPTION
**Pull request by nithinraok**
Signed-off-by: nithinraok <nithinrao.koluguri@gmail.com>

This PR adds support for the [gecko](https://gong-io.github.io/gecko/) labeling tool. Now diarization outputs *_gecko.json along with current output files for loading in gecko tool.

### CODE REVIEW

File: examples/speaker_tasks/diarization/README.md [hunk 1]
```diff
@@ -131,12 +131,13 @@ In `./demo_asr_output/`, you can check the results as below.
...
     └── my_audio1.txt
     └── my_audio1.rttm
+    └── my_audio1_gecko.json
 │
 └── ...
 ```
 
 
-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;  `*.json` files contains word-by-word json output with speaker label and time stamps.
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;  `*.json` files contains word-by-word json output with speaker label and time stamps. We do also provide json output file to view output of our diarization results in nicely formated [gecko](https://gong-io.github.io/gecko/) tool.
```
Comment (tango4j) [inline examples/speaker_tasks/diarization/README.md:140]: I suggest to change this line something like this ("nicely formatted" does not sound informative) :

We also provide  json output file for [gecko](https://gong-io.github.io/gecko/) tool where you can visualize the diarization result along with ASR output.

**Review by tango4j — CHANGES_REQUESTED** [review]
Please address the README section. Other parts look fine.

File: examples/speaker_tasks/diarization/README.md (hunk 1 above)
Comment (nithinraok) [reply examples/speaker_tasks/diarization/README.md]: updated

**Review by tango4j — APPROVED** [review]
Nicely done thanks for reflecting the comment.

**Pull request merged**